validation/kubernetes/cluster-autoscaler/main.tf

```diff
@@ -0,0 +1,21 @@
+terraform {}
+
+provider "kubernetes" {}
+
+provider "helm" {}
+
+module "cluster_autoscaler" {
+  source = "../../../modules/kubernetes/cluster-autoscaler"
+
+  providers = {
+    kubernetes = kubernetes
+    helm       = helm
+  }
+
+  cluster_name   = "foo"
```

Comment:
Nitpick: I suspect the output will be more readable if this would be `test-cluster` or `some-cluster` or `ze-cluster` or something.